### DISCUSSION TITLE
Business apps for business

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#583] [General]
What are business owners have to say about that?
Some essential programs/apps in your opinion?
Something not very well known?

**Comment by MirkosMangle** [comment]
QuickBooks is one of the main program we use for our finances.
All the expenses and incomes are registered there

**Comment by hernessteve** [comment]
Not sure how well or not very well known is [slack sms](https://clerk.chat/slack) but it is definitely worth the attention.
We used to spend a decent amount of money for communication purposes (mobile bills, calls etc) and now it is all replaced only with the slack. We use different channels for our inside teams and sending clients sms to let them know about the existing promotions or sales.

**Comment by XelaAl** [comment]
> What are business owners have to say about that? Some essential programs/apps in your opinion? Something not very well known?

I find that productivity apps like Trello, Asana, and QuickBooks are essential for task management, project planning, and financial tracking. Also, I can't imagine our working process without using an online banking app. It was a wise decision to reach out to [unify financial credit union customer service](https://www.pissedconsumer.com/company/unify-financial-credit-union/customer-service.html) and open a business account there. I should say, that modern technologies really help businesses and make work smoother.
